weather-app/src/app/home/favorite-list/favorite-item/favorite-item.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="favorite__item">
+    <i class="icon-calendar"></i>
```

Comment:
i think we need a structure and simantic for that for example

<section class="favorite__item">
  <header class="favorite__header">
      <time class="favorite__date">February 7th, 2022</time>
      <span class="favorite__city">Toronto</span>
  </header>
  <div class="favorite__body">...</div>
</section >

Reply:
<section class="favorite__item">
   <header class="favorite__header">
       <i class="icon-heart-active"></i>
       <time class="favorite__date">February 7th, 2022</time>
       <span class="favorite__city">Toronto</span>
  </header>
  <div>
  <footer>
</section>